### PR TITLE
Page2

### PR DESCRIPTION
**Pull request by aureliejr**
extremely basic html page that displays given JSON formatted poems

### CODE REVIEW

File: frontend/page2/index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                function appendData (data) {
+                    const mainCont = document.querySelector("#poem")
+
+                    for (let i = 0; i < data.length; i++){
+                        let div = document.createElement("div");
+                        div.innerHTML = `${data[i].poem}...signed ${data[i].author} // written ${data[i].date} at ${data[i].lat}&${data[i].long}`;
+                        mainCont.appendChild(div)
+                    }
+                }
+            </script>
```
Comment (nbriz) [inline frontend/page2/index.html]: let's put this code in it's own JavaScript file, maybe `/frontend/page2/display-poems.js` (something like that) && then we'll link to it in our HTML like:
```html
<script src="display-poems.js"></script>
```

**Review by nbriz — CHANGES_REQUESTED** [review]
@aurelieschool nice job! works great, but there are some changes i'm gonna need u to make before we merge this. first, in class yesterday we updated the structure of the poem objects ([documented here](https://github.com/net-art-uchicago/cta-file-sharing/issues/14)) so ur test data should now look like that. also, it looks like u've got test data in both the root directory && ur sub-folder (let's get rid of the file in the root directory && work w/the one in the sub-folder instead)

**Comment by aureliejr** [comment]
I changed things based on the comments you left + tried to fix some other issues with my code. The problem I'm running into now is either with the ".length" function or my actual JSON file, but for some reason the code seems to be detecting that there's only one object in my JSON file. I'll make the HTML/JS more interesting/better once I can get this working. Apologies for the delay.

Also: I'm not sure if this is the right way to update a PR, please let me know if there's something else I need to do

**Comment by nbriz** [comment]
@aurelieschool u don't need to close / reopen the PR, u can leave it open while u're making these changes.

what u do need to do though is `push` the edits u've made up to ur repo so that this PR gets updated w/those new changes && i can pull + test them on my end

**Comment by aureliejr** [comment]
I updated the poems file to fit the new JSON formatting. Also, the poems load now, for some reason they only work when I run the page through a local server on my computer (and not when I open the html file normally).

**Comment by aureliejr** [comment]
fixed the style and committed the other changes i had made

**Pull request merged**